src/defs/metrics.d.ts

```diff
@@ -150,10 +164,26 @@ declare module "tsdDef" {
          *
          * @param key The name of the attribute.
          * @param value The value of the attribute.
+         * @deprecated Use <code>addAnnotation</code> instead, to conform with other TSD-Metrics clients.
```

Comment:
Just call it metrics clients -- tsd is a general term and along with query log should be deprecated.